docs/network-flow-visibility.md

```diff
@@ -67,33 +72,94 @@ ConfigMap as shown below. We provide some examples for the parameter values in t
     flowExportFrequency: 5
 ```
  
-Please note that the default values for `flowPollInterval` and `flowExportFrequency` parameters are set to 5s and 12, respectively.
-`flowCollectorAddr` is a required parameter that is necessary for the Flow Exporter feature to work.
+Please note that the default values for `flowPollInterval` and `flowExportFrequency`
+parameters are set to 5s and 12, respectively. `flowCollectorAddr` is a required
+parameter that is necessary for the Flow Exporter feature to work.
+
+### IPFIX Information Elements(IEs) in a Flow Record
+
+There are 23 IPFIX IEs in each flow record, which are registered the IANA-assigned IE
+registry, the Reverse IANA-assigned IE registry and the Antrea IE registry. The
+reverse IEs are used for the flow in reverse direction when representing a
+connection. All the utilized IEs are given below:
```

Comment:
Suggestion:

There are 23 IPFIX IEs in each flow record, which are defined in the IANA-assigned IE
registry, the [Reverse IANA-assigned IE registry](https://tools.ietf.org/html/rfc5103) and the Antrea IE registry. The
reverse IEs are used to provide bi-directional information about the flow.
All the IEs used by the Antrea Flow Exporter are listed below: